src/Repositories/Payfort.php

```diff
@@ -204,6 +216,30 @@ public function isTestingFortId($fort_id)
         }
     }
 
+    public function setApplePayParams(array $params = []): self
+    {
+        if (count($params)) {
```

Comment:
no need to recalculate the apple pay data, since it's already calculated and `PayfortIntegration` class 

```suggestion
        $this->apple_pay = $apple_data
```